src/dataflow/src/render/sources.rs

```diff
@@ -787,6 +717,7 @@ fn raise_key_value_errors(KV { key, val }: KV) -> Option<Result<(Row, Row), Deco
             (Err(e), _) => Some(Err(e)),
         },
         (None, None) => None,
+        // TODO(petrosagg): this is an envelope error
```

Comment:
as in, creation of this source should fail?

Reply:
No I mean that currently we have SourceError, DecodeError, and EvalError as DataflowError variants but not a proper EnvelopeError variant